src/com/hms_networks/americas/sc/historicaldata/HistoricalDataQueueManager.java

```diff
@@ -26,11 +26,27 @@
   /** Boolean flag indicating if string history data should be included in queue data. */
   private static boolean stringHistoryEnabled = false;
 
-  /** File path for time marker */
-  private static final String timeMarkerFileName =
+  /**
+   * Boolean flag indicating if the first time marker file is the primary or secondary time marker
+   * file.
+   */
+  private static boolean useTimeTrackerFile1Primary = true;
```

Comment:
```suggestion
  private static boolean isFile1CurrTimeTrackerFile = true;
```

Reply:
this might more accuratly represent this variable. primary/secondary naming makes it seem like one file is always deemed primary rather where curr or current or even active indicate there is a time element associated with the tracking file used.